src/features/components/molecules/shadow-box/style.module.scss

```diff
@@ -0,0 +1,8 @@
+@use 'features/styles/theme/theme-mixins' as *;
+
+.box_shadow {
+    box-shadow: 0px 4px 6px -2px rgba(14, 14, 14, 0.03), 0px 12px 16px -4px rgba(14, 14, 14, 0.08);
+}
+.styled_heading{
+    max-width: 18rem;
```

Comment:
```suggestion
    max-inline-size: 18rem;
```

Reply:
Hi @konstantinosG-deriv  I have fixed this 🙏